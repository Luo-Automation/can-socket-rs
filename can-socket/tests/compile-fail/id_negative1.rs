fn main() {
	let _id = can_socket::can_id!(-2);
}
