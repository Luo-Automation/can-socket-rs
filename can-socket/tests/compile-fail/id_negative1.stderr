error[E0600]: cannot apply unary operator `-` to type `u32`
 --> tests/compile-fail/id_negative1.rs:2:32
  |
2 |     let _id = can_socket::can_id!(-2);
  |                                   ^^ cannot apply unary operator `-`
  |
  = note: unsigned values cannot be negated
